test_runner/fixtures/zenith_fixtures.py

```diff
@@ -517,6 +518,23 @@ def __exit__(self, exc_type, exc, tb):
         self.stop()
 
 
+    def list_files_to_compare(self):
+        pgdata_files = []
+        for root, __file__, filenames in os.walk(self.pgdata_dir):
```

Comment:
`__file__` has a special meaning in python (all dunderscore methods or variables typically carry some additional meaning). Probably in that case you can use `dirnames` if you do not need the value you can use `_` like in Rust

Also you can skip unwanted directories by removing items from this `dirnames` variable. See an example from our performance tests: https://github.com/zenithdb/zenith/blob/main/test_runner/performance/test_perf_pgbench.py#L15